src/FBO.cpp

```diff
@@ -160,21 +160,27 @@ FBO::SetTextureHandle(const GLuint aHandle,
 }
 
 void
-FBO::Bind() {
+FBO::Bind(GLenum aTarget) {
   if (!m.valid) { return; }
-  VRB_GL_CHECK(glBindFramebuffer(GL_FRAMEBUFFER, m.fbo));
+
+  VRB_GL_CHECK(glBindFramebuffer(aTarget, m.fbo));
 }
 
 void
-FBO::Unbind() {
-  VRB_GL_CHECK(glBindFramebuffer(GL_FRAMEBUFFER, 0));
+FBO::Unbind(GLenum aTarget) {
```

Comment:
Wouldn't it be better to just save the target in Bind() so you don't have to pass it in again of Unbind?

Reply:
Done